src/core/ColorPicker/ColorBuilder.tsx

```diff
@@ -72,6 +77,14 @@ export const ColorBuilder = React.forwardRef(
         }
       : { backgroundColor: hueColorString };
 
+    const opacitySliderStyle = getWindow()?.CSS?.supports?.(
+      `--selected-color: ${hueColorString}`,
+    )
+      ? {
+          '--selected-color': hueColorString,
+        }
```

Comment:
Very minor but this could be formatted in one line

```ts
{ '--selected-color': hueColorString }
```

Reply:
Fixed